integration/all/trace_dns_test.go

```diff
@@ -44,8 +44,26 @@ func newTraceDnsCmd(t *testing.T, ns string, dnsServerArgs string) *Command {
 	nslookupCmds := []string{
 		fmt.Sprintf("setuidgid 1000:1111 nslookup -type=a fake.test.com. %s", dnsServer),
 		fmt.Sprintf("setuidgid 1000:1111 nslookup -type=aaaa fake.test.com. %s", dnsServer),
-		fmt.Sprintf("setuidgid 1000:1111 nslookup -type=a nodomain.fake.test.com. %s", dnsServer),
```

Comment:
Why is this gone?

Reply:
Added later in the switch statement: 
`nslookupCmds = append(nslookupCmds, fmt.Sprintf("setuidgid 1000:1111 nslookup -type=a nodomain.fake.test.com. %s", dnsServer))`